src/client/common/variables/environment.ts

```diff
@@ -32,11 +32,7 @@ export class EnvironmentVariablesService implements IEnvironmentVariablesService
         if (!target) {
             return;
         }
-        const settingsNotToMerge = ['PYTHONPATH', this.pathVariable];
         Object.keys(source).forEach((setting) => {
-            if (settingsNotToMerge.indexOf(setting) >= 0) {
```

Comment:
I'm not sure I understand the fix here. So the problem is that we were previously skipping merging the PYTHONPATH and PATH settings?

Reply:
Yes, and then we added code to explicity merge it later,

Reply:
Doesn't this conflict with the kernelEnv and custeEditVars merging of path and python path? It's now replacing on merge and appending later.